scripts/populateDAO.ts

```diff
@@ -0,0 +1,335 @@
+import * as yargs from 'yargs';
+import chalk from 'chalk';
+import * as moment from 'moment';
+import * as fs from 'fs';
+import * as Arc from '@daostack/arc.js';
+import { VoteOptions, DAO } from '@daostack/arc.js';
+import Util from '../src/lib/util';
+import axios from "axios";
+import { ContributionRewardWrapper, GenesisProtocolWrapper } from '@daostack/arc.js';
+import HDWalletProvider from "../src/lib/truffle-hdwallet-provider";
+// tslint:disable-next-line:no-var-requires
+const Web3 = require("web3");
+import promisify = require("es6-promisify");
+// tslint:disable-next-line:no-var-requires
+const NonceTrackerSubprovider = require("web3-provider-engine/subproviders/nonce-tracker");
+
+// tslint:disable-next-line:no-var-requires
+require('dotenv').config();
+
+enum Level {
+    Success = 'success',
```

Comment:
looks like you may have used tabs in this file? lets stick to two spaces everywhere